noxfile.py

```diff
@@ -13,6 +13,7 @@
 
 test_deps = ["pytest"]
 coverage_deps = ["coverage", "pytest-cov"]
+coverage_report_deps = ["coverage", "jinja2<3.1", "gcovr==5.0"]
```

Comment:
- There is an issue with gcovr 5.1 and gcc11 coverage files, so we pin to 5.0
- Deprecated `jinja2.Markup` was removed in jinja2 3.1, and that breaks gcovr 5.0 if you try to generate html reports.

Reply:
Is there an upstream issue for that that we can link to in a comment? (To know when to remove the pinning again.)